test/e2e/e2e_suite_test.go

```diff
@@ -56,4 +56,5 @@ var _ = Describe("Fin", func() {
 	Context("verification", Label("verification"), Label("misc"), Ordered, verificationTestSuite)
 	Context("delete incremental backup", Label("delete-incremental-backup"), Label("misc"), Ordered,
 		deleteIncrementalBackupTestSuite)
+	Context("pvc", Label("pvc-deletion"), Ordered, pvcDeletionTestSuite)
```

Comment:
It seems the test didn't run because the workflow doesn't select the `pvc-deletion` label.
```suggestion
	Context("pvc", Label("misc"), Ordered, pvcDeletionTestSuite)
```

Reply:
Good catch! Fixed, thanks.